From 6a33952b8af18c4c0e778d93a69d97bc2adbc52e Mon Sep 17 00:00:00 2001
From: antirez <antirez@gmail.com>
Date: Fri, 14 Apr 2017 17:53:11 +0200
Subject: [PATCH 04/19] Test: fix, hopefully, false PSYNC failure like in issue
 #2715.

And many other related Github issues... all reporting the same problem.
There was probably just not enough backlog in certain unlucky runs.
I'll ask people that can reporduce if they see now this as fixed as
well.
---
 tests/integration/replication-psync.tcl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/integration/replication-psync.tcl b/tests/integration/replication-psync.tcl
index 3a41ceb2..308d63d0 100644
--- a/tests/integration/replication-psync.tcl
+++ b/tests/integration/replication-psync.tcl
@@ -110,7 +110,7 @@ foreach diskless {no yes} {
     test_psync {no reconnection, just sync} 6 1000000 3600 0 {
     } $diskless 0
 
-    test_psync {ok psync} 6 1000000 3600 0 {
+    test_psync {ok psync} 6 100000000 3600 0 {
         assert {[s -1 sync_partial_ok] > 0}
     } $diskless 1
 
-- 
2.13.1

